From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Mon, 4 Jun 2012 16:18:52 +0200
Subject: Reduce udevadm settle timeout to 10 seconds

This isn't a proper fix but it will make virt-manager at least start.

Closes: #663931
Index: libvirt-1.1.4/src/util/virutil.c
===================================================================
--- libvirt-1.1.4.orig/src/util/virutil.c	2013-11-12 11:48:38.575866998 -0600
+++ libvirt-1.1.4/src/util/virutil.c	2013-11-12 11:48:38.563866998 -0600
@@ -1444,7 +1444,7 @@
 void virFileWaitForDevices(void)
 {
 # ifdef UDEVADM
-    const char *const settleprog[] = { UDEVADM, "settle", NULL };
+    const char *const settleprog[] = { UDEVADM, "settle", "--timeout=10", NULL };
 # else
     const char *const settleprog[] = { UDEVSETTLE, NULL };
 # endif
